src/glassflow/pipelines.py

```diff
@@ -240,3 +232,24 @@ def consume_failed(self) -> operations.ConsumeFailedResponse:
                                      http_res)
 
         return res
+
+    def _get_headers(
+            self, request: dataclasses.dataclass,
+            req_content_type: Optional[str] = None
+    ) -> dict:
+
+        headers = utils.get_req_specific_headers(request)
+        headers['Accept'] = 'application/json'
+        headers['Gf-Client'] = (
+            f'{self.glassflow_client.glassflow_config.user_agent} / '
+            f'{self.glassflow_client.glassflow_config.sdk_version}')
```

Comment:
```suggestion
        headers['Gf-Client'] = (
            f'{self.glassflow_client.glassflow_config.user_agent}/'
            f'{self.glassflow_client.glassflow_config.sdk_version}')
```
What do you think to pass `glassflow-python-sdk/0.0.2` instead of `glassflow-python-sdk / 0.0.2`?

Reply:
I have no preference, I've removed the spaces 👍